haas/ext/switches/brocade.py

```diff
@@ -60,33 +60,41 @@ def session(self):
     def disconnect(self):
         pass
 
-    def apply_networking(self, action):
+    def modify_port(self, port, channel, network_id):
         """ Apply a NetworkingAction to the switch.
 
         Args:
             action: NetworkingAction to apply to the switch.
         """
-        interface = action.nic.port.label
-        channel = action.channel
+        # XXX: We ought to be able to do a Port.query ... one() here, but
```

Comment:
Replace XXX with FIXME? I suggest open an issue about this since it can be orthogonal